<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [MobilAppAutomation]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="utils.RetryListener"/>
  </listeners>
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="sZephyr_testcases.TC01_Accounts_Info">
        <methods>
          <include name="afterMethod"/>
          <include name="afterTest"/>
          <include name="startTestCase"/>
          <include name="removerepair"/>
          <include name="afterSuite"/>
          <include name="beforeTest"/>
          <include name="beforeMethod"/>
          <include name="beforeSuite"/>
        </methods>
      </class> <!-- sZephyr_testcases.TC01_Accounts_Info -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [MobilAppAutomation] -->
